app/src/Model/Meme.php

```diff
@@ -124,11 +124,11 @@ public function createMeme(){
             $status = $response->getStatusCode();
             $body = json_decode($response->getBody(), true);
```

Comment:
`$response->getBody()->getContents()`, otherwise, `$body` will not contain the data you want